src/app/effects/book.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { EffectsTestingModule, EffectsRunner } from '@ngrx/effects/testing';
+import {TestBed, fakeAsync, tick} from '@angular/core/testing';
+import {BookEffects} from './book';
+import {GoogleBooksService} from '../services/google-books';
+import {Observable} from 'rxjs';
+import {SearchAction, SearchCompleteAction} from '../actions/book';
+import {Book} from '../models/book';
+
```

Comment:
@abdulhaq-e I looked at that but based on the typescript documentation I believe the typeRoots declaration should include jasmine. It seems to do this in other editors. 

**This should include anything installed in @types which includes jasmine**
```json

    "typeRoots": [
      "../node_modules/@types"
    ]
```

So I think the src/test.ts fix is best for now. 

**Typescript Docs**

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types

```json
If typesRoots is specified, only packages under typeRoots will be included. For example:

{
   "compilerOptions": {
       "typeRoots" : ["./typings"]
   }
}
This config file will include all packages under ./typings, and no packages from ./node_modules/@types.

If types is specified, only packages listed will be included. For instance:

{
   "compilerOptions": {
       "types" : ["node", "lodash", "express"]
   }
}
```